pyccel/codegen/printing/ccode.py

```diff
@@ -2597,10 +2599,21 @@ def _print_AliasAssign(self, expr):
                 rhs = self._print(rhs_var)
                 return f'{lhs} = {rhs};\n'
         else:
-            lhs = self._print(lhs_address)
-            rhs = self._print(rhs_address)
+            managed_mem_lst = lhs_var.get_direct_user_nodes(lambda u: isinstance(u, ManagedMemory))
+            if managed_mem_lst:
+                managed_mem = managed_mem_lst[0]
+                managed_var = managed_mem.mem_var
```

Comment:
If you do not mind, I would name this variable `manager_var` instead of `managed_var`, because this is "The variable responsible for managing the memory" according to the docstring of `ManagedMemory`. I find that the word "managed" is quite confusing here